index.html

```diff
@@ -9,691 +9,80 @@
     dt {color: black}
     </style>
       <link href="https://www.w3.org/StyleSheets/TR/2016/W3C-ED" rel="stylesheet" type="text/css">
+    <script src="http://dev.w3.org/2009/dap/ReSpec.js/js/respec.js" class="remove"></script>
+    <script class="remove">
+      var respecConfig = {
+          specStatus: "ED",
+	  shortName: "PNG",
+	  subtitle: "Information technology — Computer graphics and image processing — Portable Network Graphics (PNG): Functional specification. ISO/IEC 15948:2003 (E)",
+	  copyrightStart: "1996",
+	  previousPublishDate: "2003-11-10",
+	  previousMaturity: "REC",
+	  edDraftURI: "https://github.com/w3c/PNG-spec",
+	  extraCSS: ["http://dev.w3.org/2009/dap/ReSpec.js/css/respec.css"],
+	  editors: [
+	      { name: "Chris Blume", url: "https://www.programmax.net", w3cid: 127631 },
+	      { name: "Pierre-Anthony Lemieux", company: "Sandflow Consulting LLC", companyURL: "https://www.sandflow.com" },
+	      { name: "Chris Lilley", url: "https://svgees.us", company: "W3C", companyURL: "https://www.w3.org" },
+	      { name: "Leonard Rosenthol", company: "Adobe Inc.", companyURL: "https://www.adobe.com" },
+	      { name: "Chris Seeger", company: "Comcast Corporation", companyURL: "https://www.comcast.com" }
+	  ],
+	  formerEditors: [
+	      { name: "Thomas Boutell" },
+	      { name: "Adam M. Costello" },
+	      { name: "David Duce" },
+	      { name: "Tom Lane" },
+	      { name: "Glenn Randers-Pehrson" },
+	      { name: "Oxford Brookes University (Second Edition)" }
```

Comment:
That is an affiliation, not a former editor

Reply:
Fixed.